Source/CarthageKit/Xcode.swift

```diff
@@ -787,7 +786,17 @@ public func buildDependencyProject(dependency: ProjectIdentifier, rootDirectoryU
 			}
 		}
 
-		if !NSFileManager.defaultManager().createSymbolicLinkAtURL(dependencyBinariesURL, withDestinationURL: rootBinariesURL, error: &error) {
+		// The relative path to this dependency's Carthage/Build folder, from
+		// the root.
+		let dependencyBinariesRelativePath = dependency.relativePath.stringByAppendingPathComponent(CarthageBinariesFolderPath)
+		let componentsForGettingTheHellOutOfThisRelativePath = Array(count: dependencyBinariesRelativePath.pathComponents.count - 1, repeatedValue: "..")
```

Comment:
This might also be the first and only case I’ve seen where `Repeat<T>` might apply. I don’t see any concrete benefits to using it over `Array` here, however, at least not relative to the cost of “what the hell is `Repeat`?”.